src/main/webapp/schema/schema.json

```diff
@@ -1082,7 +1204,7 @@
         },
         "name": {
           "type": "string",
-          "pattern": "^[^<>\\:\"\\/\\\\|\\?\\*]+$"
```

Comment:
@vtsvetkov-splunk Why did you change this line?

Reply:
I got something like "invalid escape character" when I tried to validate something with this schema

Reply:
I used ajv json schema validator that complained about this regexp.
```
SyntaxError: Invalid regular expression: /^[^<>\:"\/\\|\?\*]+$/: Invalid escape
```
<img width="550" alt="image" src="https://github.com/splunk/addonfactory-ucc-base-ui/assets/142901247/7d22d9d1-8786-436e-a13e-a2e7ee0b0591">

I removed the escape character to be able to run validator. 
<img width="549" alt="image" src="https://github.com/splunk/addonfactory-ucc-base-ui/assets/142901247/e76e2b82-2bef-40d5-87d2-44768e87ae7e">